src/renderers/shaders/ShaderLib/depth_vert.glsl

```diff
@@ -12,10 +12,18 @@ void main() {
 
 	#include <skinbase_vertex>
 
+	#ifdef USE_DISPLACEMENTMAP
```

Comment:
What do you mean? I thought mrdoob is the one with the ability to squash a pull request if he chooses to merge.